### DISCUSSION TITLE
DONE: Parsing of "Shots"?

### DISCUSSION THREAD

**Discussion by tin2tin** [tin2tin/Blender_Screenwriter#41] [General]
It's not included in the official formatting rules of fountain - notes should not be included in prints/previews. However here's an example of parsing of notes with a shot tag interpreted as "shots": https://slugline.co/blog/shotlists

This is where the notes gets parsed:
https://github.com/tin2tin/Blender_Screenwriter/blob/master/fountain.py#L163

Question is what should "shots" be converted to during "text to strips" processing? A camera in the scene(previz friendly) or as a Scene(storyboard friendly, but will be messy to implement the scene options to shots too)?

**Comment by tin2tin** [comment]
Shot parsing added: https://mobile.twitter.com/tintwotin/status/1408733158762291201

**Comment by gabrielmontagne** [comment]
This is a cool feature. It's great that you went for cameras. That makes,
to me, a lot of sense. 👍

**Comment by tin2tin** [comment]
@gabrielmontagne There are some bugs still. Regex is driving me nuts. Thanks for coming by.  I'm might try to give the nodes-stuff a shot at some point, if we can come up with a useful design.

**Comment by tin2tin** [comment]
Fixed some bugs and added Insert Shot button, Storyboarding friendly camera settings and scene strips for each new camera.
![shots2](https://user-images.githubusercontent.com/1322593/123524538-450b3680-d6cb-11eb-96a3-dbb312d89e34.gif)
